src/pages/state-channels/[id].vue

```diff
@@ -6,6 +6,10 @@
   <page-header>
```

Comment:
https://github.com/aeternity/aescan/assets/15363559/a364f40a-6f4f-4a8e-9283-8915582ab480

Does not seems to work here

Reply:
My bad, I must have broke it during pre-push cleanup

Reply:
Thank you, fixed